src/config.ts

```diff
@@ -3,9 +3,13 @@ import path from 'path';
 import yaml from 'js-yaml';
 import Ajv from 'ajv';
 
+export enum LanguageOptions {
+    FLOW = 'flow',
+    TYPESCRIPT = 'typescript',
+}
```

Comment:
nit: enum is fine i guess, but I think a union would be more idiomatic 

<img width="821" alt="Screenshot 2024-02-15 at 8 57 36 AM" src="https://github.com/Yelp/dataloader-codegen/assets/1590756/b496725d-4f20-4264-bc8c-d8e81c049172">

Reply:
(also note the `LanguageOptions` -> `LanguageOption` (singular), since there's only one one option being decided here (the target language)